controller/src/main/java/io/pravega/controller/store/InMemoryScope.java

```diff
@@ -147,4 +151,32 @@ public String getName() {
     @Override
     public void refresh() {
     }
+
+    @Override
+    public CompletableFuture<Pair<List<String>, String>> listKeyValueTables(int limit, String continuationToken, Executor executor) {
+        if (kvTablesMap == null) {
+            return Futures.failedFuture(StoreException.create(StoreException.Type.DATA_NOT_FOUND, this.scopeName));
+        }
+        List<String> sortedKVTablesList = kvTablesMap.keySet().stream().collect(Collectors.toList());
+        int start = 0, end = 0;
+        if (!continuationToken.isEmpty()) {
+            start = Integer.parseInt(continuationToken);
+        }
+
+        if ((start + limit) >= sortedKVTablesList.size()) {
```

Comment:
This can be simplified in a one-liner: `end = ((start + limit) >= sortedKVTablesList.size()) ? sortedKVTablesList.size() : start + limit;`

Reply:
done